planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -634,6 +636,11 @@ void planning_scene_monitor::PlanningSceneMonitor::excludeRobotLinksFromOctree()
       if (h)
         link_shape_handles_[links[i]].push_back(std::make_pair(h, j));
     }
+    if(!warned && ((ros::Time::now() - start) > ros::Duration(30.0)))
+    {
+      ROS_WARN_STREAM_NAMED("PlanningSceneMonitor::excludeRobotLinksFromOctree()", "It is likely there are too many vertices in collision geometry");
+      warned = true;
```

Comment:
```
#include <ros/ros.h>

int main(int argc, char** argv){
   ros::init(argc, argv, "mytest");
   ros::NodeHandle nh;
   int i=0;
   ros::Time t0= ros::Time::now();
   while(i++ < 50){
      if(ros::Time::now() < t0)
         std::cout << "impossible" << std::endl;
   }
   std::cout << ros::Time::now() - t0 << std::endl;
   return 0;
}
```
over here this prints:
```
$ rosrun mytest mytest_time
0.000005478
```
I'm quite sure this is neglectable.
Also 30 seconds are much more easy to interpret than 30.000.000 vertices.
I'm not even sure which order of number would make sense for such a comparison...